### PR TITLE
Improvement/AppStore: Remove seeding from app-store-cli.

### LINKED ISSUE #8375: [CAL-1525] Stop Adding seeds for new apps
<sub>From [SyncLinear.com](https://synclinear.com) | [CAL-1525](https://linear.app/calcom/issue/CAL-1525/stop-adding-seeds-for-new-apps)</sub>

### PR DESCRIPTION
**Pull request by hariombalhara**
## What does this PR do?

Fixes #8375

### Demo
[Loom](https://www.loom.com/share/ffbdf4644b134faba348fd6170a4fe68)

By removing the seeder the app installation flow locally becomes the same as in production. In production, an app isn't enabled by default and an admin needs to enable the app.

- It also informs during app creation the steps that need to be taken to enable the app
- If user goes to the app page without enabling it, he would see a message telling him that the app is disabled and it can be enabled from settings/admin/apps. This is applicable to NON PRODUCTION environment. In Production if an App is disabled, it would 404 like it doesn't exist.
   Screenshot 2023-05-10 at 9 35 26 AM
- It also deprecates doing any changes in `seed-app-store.config.json` and `seed-app-store.ts`

**Environment**: Production

## Type of change

- Improvement

## How should this be tested?
- [x] Use AppStore Cli to create an app. Visit the app page and test that it shows the app is disabled. Go to settings/admin/apps and enable it.

### CODE REVIEW

File: apps/web/pages/apps/[slug]/index.tsx [hunk 1]
```diff
@@ -32,7 +33,18 @@ const sourceSchema = z.object({
   }),
 });
 
-function SingleAppPage({ data, source }: inferSSRProps<typeof getStaticProps>) {
+function SingleAppPage(props: inferSSRProps<typeof getStaticProps>) {
+  if (props.isAppDisabled) {
```
Comment (hariombalhara) [inline apps/web/pages/apps/[slug]/index.tsx:37]: Add a NODE_ENV check here as well. It is to be shown only in Dev environment.

File: packages/app-store-cli/src/components/AppCreateUpdateForm.tsx [hunk 2]
```diff
@@ -242,6 +240,10 @@ export const AppForm = ({
                 <Text color="green">Publisher Email: </Text>
                 <Text>{email}</Text>
               </Box>
+              <Text bold>
+                Next Step: Enable the app from http://localhost:3000/settings/admin/apps as admin user (Email:
+                admin@example.com, Pass: ADMINadmin2022!)
```
Comment (hariombalhara) [inline packages/app-store-cli/src/components/AppCreateUpdateForm.tsx:245]: It's the password from seed, so no harm in showing here for quick access.

File: apps/web/pages/apps/[slug]/index.tsx [hunk 3]
```diff
@@ -80,29 +95,43 @@ export const getStaticPaths: GetStaticPaths<{ slug: string }> = async () => {
 export const getStaticProps = async (ctx: GetStaticPropsContext) => {
   if (typeof ctx.params?.slug !== "string") return { notFound: true };
 
-  const app = await prisma.app.findUnique({
-    where: { slug: ctx.params.slug.toLowerCase() },
+  const appMeta = await getAppWithMetadata({
+    slug: ctx.params?.slug,
   });
 
-  if (!app) return { notFound: true };
```
Comment (hariombalhara) [inline apps/web/pages/apps/[slug]/index.tsx:87]: Renamed it to appFromDb

File: apps/web/pages/apps/[slug]/index.tsx [hunk 4]
```diff
@@ -80,29 +95,43 @@ export const getStaticPaths: GetStaticPaths<{ slug: string }> = async () => {
...
+  const appFromDb = await prisma.app.findUnique({
+    where: { slug: ctx.params.slug.toLowerCase() },
+  });
 
-  const singleApp = await getAppWithMetadata(app);
+  const isAppAvailableInFileSystem = appMeta;
+  const isAppDisabled = isAppAvailableInFileSystem && (!appFromDb || !appFromDb.enabled);
+
+  if (process.env.NODE_ENV !== "production" && isAppDisabled) {
```
Comment (hariombalhara) [inline apps/web/pages/apps/[slug]/index.tsx:109]: In Non production env we show it as disabled othewise 404

**Review by hariombalhara — COMMENTED** [review]
Self review done

**Review by joeauyeung — APPROVED** [review]
LGTM. This is a really good idea.

**Pull request merged**